### PR TITLE
Kenny/futr 886 update agent one to cat one in cli

### PR DESCRIPTION
**Pull request by adeleke5140**
This PR does the ff:

1. updates `core` to use `zod` as a dep instead of peer-dep
2. updates docs to `cat-one` instead of `agent-one`
3.  add function to retrieve .env value `env` from environment so we can run `OPENAI_API_KEY=sk-key mastra serve`

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -65,10 +65,8 @@
     "inngest": "3.22.3",
     "radash": "^12.1.0",
     "sift": "^17.1.3",
-    "xstate": "^5.19.0"
-  },
-  "peerDependencies": {
-    "zod": "3.23.7"
+    "xstate": "^5.19.0",
+    "zod": "^3.23.8"
```
Comment (Joshuafolorunsho) [inline packages/core/package.json]: We probably want to get rid of the `peerDependecies` and add to the dependency.

File: packages/core/package.json [hunk 2]
```diff
@@ -65,10 +65,8 @@
     "inngest": "3.22.3",
     "radash": "^12.1.0",
     "sift": "^17.1.3",
-    "xstate": "^5.19.0"
-  },
-  "peerDependencies": {
-    "zod": "3.23.7"
+    "xstate": "^5.19.0",
```
Comment (Joshuafolorunsho) [inline packages/core/package.json]: Not sure why we are adding xstate as peerDependencies. I don’t think we should prompt the users of Mastra to install xstate.

**Review by Joshuafolorunsho — CHANGES_REQUESTED** [review]

File: packages/core/package.json (hunk 2 above)
Comment (adeleke5140) [reply packages/core/package.json]: i see you've modified this on your PR so I am gonna be reverting

**Review by taofeeq-deru — APPROVED** [review]

**Review by Joshuafolorunsho — APPROVED** [review]
LGTM.

**Pull request merged**